turbolinks/checkout.html

```diff
@@ -0,0 +1,311 @@
+<!doctype html>
+	<html lang="en">
+	<head>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+		<meta name="description" content="">
+		<meta name="author" content="Mark Otto, Jacob Thornton, and Bootstrap contributors">
+		<meta name="generator" content="Hugo 0.84.0">
+		<title>PixieBrix Turbolinks</title>
+
+		<link rel="canonical" href="https://getbootstrap.com/docs/5.0/examples/dashboard/">
+
+		<!-- Bootstrap core CSS -->
+		<link href="./assets/dist/css/bootstrap.min.css" rel="stylesheet" data-turbolinks-track="reload">
+		<meta name="turbolinks-visit-control" content="reload">
+
+		<style>
+		.bd-placeholder-img {
+			font-size: 1.125rem;
+			text-anchor: middle;
+			-webkit-user-select: none;
+			-moz-user-select: none;
+			user-select: none;
+		}
+
+		@media (min-width: 768px) {
+			.bd-placeholder-img-lg {
+				font-size: 3.5rem;
+			}
+		}
+	</style>
+
+
+	<!-- Custom styles for this template -->
+	<link href="./dashboard.css" rel="stylesheet" data-turbolinks-track="reload">
+	<link href="./form-validation.css" rel="stylesheet" data-turbolinks-track="reload">
+</head>
+<body>
+
+	<header class="navbar navbar-dark sticky-top bg-dark flex-md-nowrap p-0 shadow">
```

Comment:
Let's add `data-turbolinks-permanent` to the header too. I think you'll need to give it an `id` attribute when you do that too